src/firebase/config.js

```diff
@@ -0,0 +1,10 @@
+import * as firebase from 'firebase';
+
+const firebaseConfig = {
+	// as for now, i add the full file at the Trello Board.
+	// but defentily should watch again Andrei env videos again..
```

Comment:
so what do you suggested to put in this file in the meanwhile?
the actual fb credentials? 
bottom line theres no credit card attached to the user...

Reply:
@YogiHa I would like to avoid putting credentials to a javascript file, I think we need to add .env file store credentials in there and import .env file to config.js. The downside is that .env file won't be added to GitHub, and contributors would need to create .env file locally.

Reply:
ok, 
so let me know what to put here instead of the comment pls..
and BTW, the crash you were prt screen above seems like you haven't attached the config file..